packages/botonic-plugin-text-classification/src/model/model-info.ts

```diff
@@ -11,7 +12,10 @@ export class ModelInfo {
 
   constructor(readonly locale: Locale, readonly uri: string) {
     this.config = axios({ url: `${this.uri}/${CONFIG_FILENAME}` })
-    this.model = loadLayersModel(`${this.uri}/${MODEL_FILENAME}`)
+    // fetchFunc is defined to replace window.fetch to work in node environment
```

Comment:
please create a ticket to unify both ModelInfo classes.
 
* create a single class template at botonic-nlp.  Just modify this
```
 export class GenericModelInfo<T> {
  private readonly config: AxiosPromise<T>
async getConfig(): Promise<T> {
```
* Move the constants CONFIG_FILENAME & MODEL_FILENAME to be static fields of GenericModelInfo. constants should be as close as possible to the class that use them. They only need to be in a centralized placed when they need to be configured in a centralized way through env vars. even in that case, there should be a class which encapsulates them duplicating them and encapsulating them
* at each ner & textclassifer, 
export class NerModelInfo extends GenericModelInfo<NerConfig> {}